mingw-build.cmd

```diff
@@ -7,17 +7,17 @@ REM the PATH environment needs to include 7z.exe, mingw32-make.exe and a working
 REM installation.
 set "PATH=C:\Program Files\7-Zip;C:\TDM-GCC-32\bin;%PATH%"
 
-set "CMAKE_URL=file://C:/Users/joeub/Downloads/cmake-3.16.0-win32-x86.zip"
-set "CMAKE_FILENAME=cmake-3.16.0-win32-x86.zip"
-set "CMAKE_DIRNAME=cmake-3.16.0-win32-x86"
-set "BZIP2_URL=file://C:/Users/joeub/Downloads/bzip2-1.0.8.tar.gz"
+set "CMAKE_URL=https://github.com/Kitware/CMake/releases/download/v3.17.3/cmake-3.17.3-win32-x86.zip"
+set "CMAKE_FILENAME=cmake-3.17.3-win32-x86.zip"
+set "CMAKE_DIRNAME=cmake-3.17.3-win32-x86"
```

Comment:
Version changes will require testing. Everything else looks good to me.

Reply:
It builds fine. What else should be tested? I can downgrade to 3.16.0 if you want.